### PR TITLE
Remove Windows simulator command for memory_attributes

### PR DESCRIPTION
**Pull request by intel-liudean**
# Existing Sample Changes
## Description

Currently running simulator for this design on Windows will hang, so we are removing Windows simulator command for this design.

## Type of change
Documentation Change

### CODE REVIEW

**Review by sophimao — COMMENTED** [review]
Do you need to remove the fpga_sim target from src/CMakeLists.txt or is it okay to keep it?

**Comment by intel-liudean** [comment]
I think we might need to modify src/CMakeLists.txt so that one can only run fpga_sim target on Linux not on Windows. I will need to confirm with Yohann.

**Review by yuguen — COMMENTED** [review]
I think it's also worth modifying the CMakeLists.txt to have something like:

```
if(UNIX)
... sim target definition
endif()
```

File: DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes/src/CMakeLists.txt [hunk 1]
```diff
@@ -1,9 +1,12 @@
 set(SOURCE_FILE memory_attributes.cpp)
 set(TARGET_NAME memory_attributes)
 set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
-set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
 set(FPGA_TARGET ${TARGET_NAME}.fpga)
 
+if(UNIX)
```
Comment (yuguen) [inline DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes/src/CMakeLists.txt:6]: It's more than that, you can see below there is an entire "paragraph" defining the simulation rules:

```
###############################################################################
### FPGA Simulator
###############################################################################
# To compile in a single command:
#    icpx -fsycl -fintelfpga -Xssimulation -DFPGA_SIMULATOR mem_channel.cpp -o mem_channel.fpga_sim
# CMake executes:
#    [compile] icpx -fsycl -fintelfpga -Xssimulation -DFPGA_SIMULATOR -o mem_channel.cpp.o -c mem_channel.cpp
#    [link]    icpx -fsycl -fintelfpga -Xssimulation  mem_channel.cpp.o -o mem_channel.fpga_sim
add_executable(${SIMULATOR_TARGET} ${SOURCE_FILE})
target_include_directories(${SIMULATOR_TARGET} PRIVATE ../../../../include)
set_target_properties(${SIMULATOR_TARGET} PROPERTIES COMPILE_FLAGS "${SIMULATOR_COMPILE_FLAGS}")
set_target_properties(${SIMULATOR_TARGET} PROPERTIES LINK_FLAGS "${SIMULATOR_LINK_FLAGS}")
add_custom_target(fpga_sim DEPENDS ${SIMULATOR_TARGET})
```

You need to gate that as well

Comment (intel-liudean) [reply DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes/src/CMakeLists.txt:6]: Ok I see, will do.

**Review by yuguen — APPROVED** [review]
LGTM

**Review by sophimao — APPROVED** [review]

**Pull request closed without merging**